src/main/resources/db/migration/V2_07__add_shedlock_tables.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE shedlock (
```

Comment:
shed_lock may be

Reply:
Yes, needs to be shedlock.

Reply:
I had a look could not find a  way to change the table name, without hassel